src/ReportManager/DetailsSection/index.js

```diff
@@ -216,7 +221,7 @@ const DetailsSection = ({
         : null}
     </div>
 
-    {!!formSchema && !isNewFormSchema && <Form
+    {!!formSchema && !isNewFormSchemaSupportEnabled && <Form
```

Comment:
Hmm this is weird. Remember that we need both implementations to work at the same time. What I read here is that depending on the feature flag we use the old or the new Form component. But what we want is that if the feature flag is off, we only use the old version. If the feature flag is on, we use both and we depend of the schema draft to decide which Form component to use.

Reply:
I have a question regarding check if a schema comes from EFB or not, I remember that using a condition based on the draft property caused some discussions about if that was the best way to decide, also a suggestion was check which endpoint was used to fetch the data? not sure, Can you point me on the right direction? I'm a little bit lost here   

@luixlive @JoshuaVulcan